### PR TITLE
OSDOCS-11890 Documenting installing Nutanix with additional disks

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-11890

Link to docs preview:
https://81769--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installation-config-parameters-nutanix#installation-configuration-parameters-additional-nutanix_installation-config-parameters-nutanix

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by yanhua121** [comment]
In the install-config.yaml configuration, both the "dataDisks" and "gpus" are currently only supported for "compute" nodes, but not for control-plane nodes. So the "dataDisks" configuration parameters should documented as:
compute:
  platform:
    nutanix:
      dataDisks:
       ....

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
...
+|Optional. The name of the storage container resource in Prism Central.
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        dataSourceImage:
+          referenceName:
+|Optional. The identifier of the storage resource configured in the failure domain. If you use this identifier, you must configure a matching failure domain storage resource. For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3199]: Should be:

Optional. The identifier of the data source image in the failure domain. If you use this identifier, you must configure a matching dataSourceImage (with the same referenceName) in each failure domain the compute nodes will spread out. For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
...
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        dataSourceImage:
+          uuid:
+|The UUID of the storage resource in Prism Central. This value is required if you attach an additional storage resource to virtual compute machines.
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3208]: Should be:

The UUID of the data source image in Prism Central.

File: modules/installation-configuration-parameters.adoc [hunk 3]
```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
...
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        deviceProperties:
+          adapterType:
+|The adapter type of the storage resource. If the disk type is "Disk", valid values are "SCSI", "IDE", "PCI", "SATA" or "SPAPR".
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3217]: The adapter type of the disk address. ....

File: modules/installation-configuration-parameters.adoc [hunk 4]
```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
...
+|The index of the storage resource. The device index for disks that share the same adapter type should start at 0 and increase consecutively. The default value is 0. For each virtual machine, the `Disk.SCSI.0` and `CDRom.IDE.0` indices are reserved. If you use the `Disk.SCSI` or `CDRom.IDE` disk and adapter types, the `deviceIndex` should start at 1.
+|Non-negative integer, including 0.
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        deviceProperties:
+          deviceType:
+|The device type of the storage resource. Valid values are "Disk" and "CDRom". The default value is "Disk".
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3236]: Should be:

The disk device type. Valid values are "Disk" and "CDRom".

File: modules/installation-configuration-parameters.adoc [hunk 5]
```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
...
+If the disk type is "CDRom", valid values are "IDE" or "SATA".
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        deviceProperties:
+          deviceIndex:
+|The index of the storage resource. The device index for disks that share the same adapter type should start at 0 and increase consecutively. The default value is 0. For each virtual machine, the `Disk.SCSI.0` and `CDRom.IDE.0` indices are reserved. If you use the `Disk.SCSI` or `CDRom.IDE` disk and adapter types, the `deviceIndex` should start at 1.
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3227]: The index of the disk address. The valid values are non-negative integers. The device index for disks that share the same adapter type should start at 0 and increase consecutively. The default value is 0. For each virtual machine, the `Disk.SCSI.0` and `CDRom.IDE.0` indices are reserved. If you use the `Disk.SCSI` or `CDRom.IDE` disk and adapter types, the `deviceIndex` should start at 1.
Comment

File: modules/installation-configuration-parameters.adoc [hunk 6]
```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
...
+|The size of the disk to attach to the virtual machine. The minimum size is 1Gb.
+|Quantity format, such as 100G or 100Gi. For more information on this format, see link:https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Format.
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        storageConfig:
+          diskMode:
+|The disk mode of the storage resource. Valid values are "Standard" or "Flash", and the default is "Standard".
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3253]: The disk mode. Valid values are "Standard" or "Flash", and the default is "Standard".

File: modules/installation-configuration-parameters.adoc [hunk 7]
```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
...
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        storageConfig:
+          storageContainer:
+            referenceName:
+|Optional. The reference name of the storage container in the failure domain. If you use this identifier, you must configure a matching failure domain storage resource. For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3273]: Optional. The reference name of the storage container in the failure domain. If you use this, you must configure a matching storageContainer (with the same reference name) in each failure domain the compute nodes will spread out.  For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.

**Comment by bscott-rh** [comment]
@sgaoshang  PTAL at this PR for additional disks on Nutanix. ty

**Review by lpettyjo — COMMENTED** [review]
LGTM!

**Pull request merged**